components/molecules/Quote.js

```diff
@@ -0,0 +1,46 @@
+import PropTypes from "prop-types";
+
+/**
+ * Quote component
+ */
+export function Quote(props) {
+  return (
+    <div
+      title="Quote"
+      className="border-l-4 text-sm md:text-p pl-4 leading-normal font-body"
+    >
+      {props.html === undefined ? (
+        <div>{props.children}</div>
+      ) : (
+        <div dangerouslySetInnerHTML={{ __html: props.html }} />
+      )}
+      <p className="text-gray-500 pt-4"> — {props.author}</p>
+    </div>
+  );
+}
+
+Quote.propTypes = {
+  /**
+   * Option for styling component
+   */
```

Comment:
Think you forgot to include this prop here in the code. Note that the size of the qoute should be standard its just the container size we are altering. So look at adding a max width for qoute